benchmarks/nonlinear_channel_flow/simple_nonlinear.cc

```diff
@@ -310,7 +313,7 @@ namespace aspect
                              "List of viscosity prefactors, $A$, for background mantle and compositional fields, "
                              "for a total of N+1 values, where N is the number of compositional fields. "
                              "If only one values is given, then all use the same value. "
-                             "Units: $Pa^{-n_{dislocation}} m^{n_{dislocation}/m_{dislocation}} s^{-1}$");
+                             "Units: \\si{\\pascal}$^{-n_{dislocation}}$ \\si{\\meter}$^{n_{dislocation}/m_{dislocation}}$ \\si{\\per\\second}");
```

Comment:
There ought to be a better way to do this. I've looked at the siunitx manual (https://mirror.las.iastate.edu/tex-archive/macros/latex/contrib/siunitx/siunitx.pdf). The only thing I can find is subscripts on p. 40. Do you see anything better? In any case, while you're there, let's change 
```suggestion
                             "Units: \\si{\\pascal}$^{-n_\\text{dislocation}}$ \\si{\\meter}$^{n_\\text{dislocation}/m_\\text{dislocation}}$ \\si{\\per\\second}");
```
to make sure that it shows up as text, rather than in wide-spaced italics.

Reply:
I've added the \\text. 
I can't see a good way to deal with these expressions. I tried a couple of things, but the subscripted-superscript is particularly nasty, and siunitx didn't accept any of the iterations I tried.

Reply:
OK, thanks for trying!